frontend/public/components/storage/shared.ts

```diff
@@ -23,31 +28,32 @@ type ProvisionerAccessModeMapping = {
   [provisioner: string]: AccessModeMapping;
 };
 
-// See https://kubernetes.io/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes for more details
+// See https://kubernetes.io/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes and
+// https://docs.openshift.com/container-platform/4.15/storage/understanding-persistent-storage.html for more details
 export const provisionerAccessModeMapping: ProvisionerAccessModeMapping = Object.freeze({
```

Comment:
FYI: Please verify PVC - Storage Class Provisioners below if any is missing the `ReadWriteOncePod` Access Mode?